runtimes/scala/play-framework-2.md

```diff
@@ -20,8 +20,7 @@ Refer to the page [Deploy an application on Clever Cloud](/clever-cloud-overview
 ## Necessary information
 
 * The application must be located at the **root** of the git repository.
-* Play 2.4 need **Java 8** to work. You need to [select Java 8](https://www.clever-cloud.com/doc/java/select-java-version/)
-for your application.
+* Your application may need **Java 8** to work (for example Play 2.4 need it). You can do it [this way](https://www.clever-cloud.com/doc/java/select-java-version/).
```

Comment:
Starting from play 2.4, your application needs java 8 to run, please read [select the java version](https://www.clever-cloud.com/doc/java/select-java-version/) for more information.